front_end/src/components/payment/BookingInfo.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+
+// import "typeface-roboto";
+import { Grid } from "@material-ui/core";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Paper from "@material-ui/core/Paper";
+import { withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import { connect } from "react-redux";
+import moment from "moment";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+
+export class BookingInfo extends Component {
+  render() {
+    const time = moment().format("dddd, MMMM Do YYYY");
+    // const individualHotelData = this.props.individualHotelData.individualHotelData;;
+    const { individualHotelData } = this.props.individualHotelData;
+    // console.log(
+    //   "booking info query ",
+    //   JSON.stringify(this.props.query.searchQuery)
+    // );
+    // console.log("booking info query ", this.props.query.searchQuery);
+    const query = this.props.query;
+
+    const roomSelection = this.props.roomSelection;
+
+    return (
+      <React.Fragment>
+        <br />
+        <br />
+        <Paper style={{ margin: "10%", backgroundColor: "#e3ecf7" }}>
+          <div>
+            <img src={individualHotelData.img[0]} alt="image" />
+          </div>
+          <h4
+            className="display-4 text-left col-10"
+            style={{ marginTop: "5%", fontSize: 15, fontWeight: "bold" }}
+          >
+            {individualHotelData.name}
+          </h4>
+          <h5
+            className="display-4 test-center"
+            style={{ fontSize: 15, color: "#808080" }}
+          >
+            {individualHotelData.street}, {individualHotelData.city}, 94128
```

Comment:
please make font size 14 and align the left side with the hotel name.

please also take out "94128"